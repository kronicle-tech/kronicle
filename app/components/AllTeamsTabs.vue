<template>
  <div class="my-3">
    <b-button to="/all-teams">Teams</b-button>
    <b-button to="/all-teams/components">Components</b-button>
    <b-button to="/all-teams/tech-debts">Tech Debts</b-button>
    <b-button to="/all-teams/tests">Tests</b-button>
  </div>
</template>

<script lang="ts">
import Vue from 'vue'
import { BButton } from 'bootstrap-vue'

export default Vue.extend({
  components: {
    'b-button': BButton,
  },
})
</script>
