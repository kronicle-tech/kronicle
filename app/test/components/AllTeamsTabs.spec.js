import { mount } from '@vue/test-utils'
import AllTeamsTabs from '@/components/AllTeamsTabs.vue'

describe('AllTeamsTabs', () => {
  let propsData
  let wrapper
  const createWrapper = () => {
    wrapper = mount(AllTeamsTabs, { propsData })
  }

  beforeEach(() => {
    propsData = {}
  })

  afterEach(() => {
    wrapper.destroy()
    wrapper = null
  })

  test('renders the component', () => {
    createWrapper()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
