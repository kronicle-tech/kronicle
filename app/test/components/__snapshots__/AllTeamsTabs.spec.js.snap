// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllTeamsTabs renders the component 1`] = `"<div class=\\"my-3\\"><a href=\\"/all-teams\\" target=\\"_self\\" class=\\"btn my-1 btn-secondary\\"> Teams </a> <a href=\\"/all-teams/components\\" target=\\"_self\\" class=\\"btn my-1 btn-secondary\\"> Components </a> <a href=\\"/all-teams/tech-debts\\" target=\\"_self\\" class=\\"btn my-1 btn-secondary\\"> Tech Debts </a> <a href=\\"/all-teams/tests\\" target=\\"_self\\" class=\\"btn my-1 btn-secondary\\"> Tests </a></div>"`;
